TestON/drivers/common/cli/onosclidriver.py

```diff
@@ -1758,11 +1758,19 @@ def checkIntentState( self, intentsId, expectedState = 'INSTALLED' ):
             returnValue = main.TRUE
             for intents in intentsDict:
                 if intents.get( 'state' ) != expectedState:
-                    main.log.info( self.name + " : " + intents.get( 'id' ) +
-                                   " actual state = " + intents.get( 'state' )
-                                   + " does not equal expected state = "
-                                   + expectedState )
-                    returnValue = main.FALSE
+                    if intents.get( 'state' ) == "INSTALLING":
```

Comment:
True, I just realize that if the intent is stuck in INSTALLING state this function will still return TRUE and not catch a bug. Thanks! i'll modify it right away as well as add modify the docs.